meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx6ullevk-sec/provision.uuu

```diff
@@ -0,0 +1,17 @@
+uuu_version 1.2.39
```

Comment:
I think provision.uuu can be dropped for now, and we can just leverage full_image.uuu (as they are basically the same at this stage). Later on we might create a provision script just for generating the required sota credentials on rpmb/optee/sks or similar.

Reply:
I removed the commit for `provision.uuu`, and transformed `bootloader.uuu` and `full_image` to use the signed SPL as suggested by @vanmaegima